radio/src/gui/480x272/model_receiver_options.cpp

```diff
@@ -171,21 +179,19 @@ bool menuModelReceiverOptions(event_t event)
           }
           break;
 
-        case ITEM_RECEIVER_SETTINGS_SPORT_FPORT:
-          lcdDrawText(MENUS_MARGIN_LEFT, y, "F.Port");
-          reusableBuffer.hardwareAndSettings.receiverSettings.fport = editCheckBox(reusableBuffer.hardwareAndSettings.receiverSettings.fport, RECEIVER_OPTIONS_2ND_COLUMN, y, attr, event);
-          if (attr && checkIncDec_Ret) {
-            reusableBuffer.hardwareAndSettings.receiverSettings.dirty = RECEIVER_SETTINGS_DIRTY;
-          }
-          break;
-
-        case ITEM_RECEIVER_SETTINGS_FPORT2:
-          lcdDrawText(MENUS_MARGIN_LEFT, y, "F.Port2");
-          reusableBuffer.hardwareAndSettings.receiverSettings.fport2 = editCheckBox(reusableBuffer.hardwareAndSettings.receiverSettings.fport2, RECEIVER_OPTIONS_2ND_COLUMN, y, attr, event);
-          if (attr && checkIncDec_Ret) {
+        case ITEM_RECEIVER_SETTINGS_FPORT:
+        {
+          lcdDrawText(0, y, "PORT");
```

Comment:
You seem to be using STR_PROTOCOL for b&w, why hardcoding a (different) string here ?

Reply:
Sorry for my mistake